src/partest/scala/tools/partest/nest/Runner.scala

```diff
@@ -532,7 +533,9 @@ class Runner(val testFile: File, val suiteRunner: SuiteRunner, val nestUI: NestU
           case s if (p findFirstIn s).nonEmpty => for (m <- p findFirstMatchIn s) yield m group "args"
         }
       } finally src.close()
-      args.flatten map argsplitter getOrElse Nil
+      val parsed = args.flatten map argsplitter getOrElse Nil
+      // be forgiving of /* scalac: ... */
+      if (parsed.lastOption contains "*/") parsed.init else parsed
```

Comment:
Nice touch! Instead of argsplitter, you might consider tokenize:

```
@@ -521,7 +523,7 @@ class Runner(val testFile: File, val suiteRunner: SuiteRunner, val nestUI: NestU
 
   // inspect given files for tool args
   def toolArgsFor(files: List[File])(tool: String, split: Boolean = true): List[String] = {
-    def argsplitter(s: String) = if (split) words(s) filter (_.nonEmpty) else List(s)
+    import scala.tools.cmd.CommandLineParser.tokenize
     def argsFor(f: File): List[String] = {
       import scala.util.matching.Regex
       val p    = new Regex(s"(?:.*\\s)?${tool}:(?:\\s*)(.*)?", "args")
@@ -532,7 +534,9 @@ class Runner(val testFile: File, val suiteRunner: SuiteRunner, val nestUI: NestU
           case s if (p findFirstIn s).nonEmpty => for (m <- p findFirstMatchIn s) yield m group "args"
         }
       } finally src.close()
-      args.flatten map argsplitter getOrElse Nil
+      val parsed = args.flatten.map(tokenize).getOrElse(Nil)
+      // be forgiving of /* scalac: ... */
+      if (parsed.lastOption contains "*/") parsed.init else parsed
```